server/files/stylesheets/docs.css

```diff
@@ -38,6 +38,17 @@ p:last-child {
 footer.footer {
   padding: 4em 1em !important;
 }
+.footer .ui.link.list > .item svg {
+  width: 0.7em;
+  height: auto;
+  margin-left:0.2em;
```

Comment:
Leave a space between key and value (yes, i am nitpicking, but so is prettier.io / linting
```suggestion
  margin-left: 0.2em;
```

Reply:
I have seen most of the open source project use prettier.